site/content/docs/5.3/examples/footers/index.html

```diff
@@ -13,12 +13,12 @@ <h1 class="visually-hidden">Footers Boosted example</h1>
 
   <div class="b-example-divider"></div>
 
-  {{< orange-footer mandatory=true id="1" >}}
+  {{< orange-footer mandatory=true in_examples=true id="1" >}}
```

Comment:
Instead of using the source for the naming, can't we use the purpose instead for better reusability? So instead of `in_examples`, use `show_footer_active_link` (or a better naming).